ui/src/frontend/index.ts

```diff
@@ -124,6 +124,8 @@ function setupContentSecurityPolicy() {
       'ws://127.0.0.1:8037', // For the adb websocket server.
       'https://*.google-analytics.com',
       'https://*.googleapis.com', // For Google Cloud Storage fetches.
+      `ws://${window.location.hostname}:9001`,
```

Comment:
A couple of things:
- Why do we need this? It's unlikely that a trace processor instance will be hosted on ui.perfetto.dev for instance.
- They should probably go into the `rpcPolicy` list above anyway.